.gitignore

```diff
@@ -66,3 +66,6 @@ server_config.json
 work_dir
 artifact_dir
 build-tools
+
+#nix
```

Comment:
i assume this can be removed now

Reply:
This is just to keep `git status` clean, it's not related to `filterSource`.